app.js

```diff
@@ -0,0 +1,84 @@
+var path = require('path');
+var fs = require('fs');
+var express = require('express');
+var Router = require('react-router');
+
+var indexStaticWatcher = require('./lib/index-static-watcher').create();
+var PORT = process.env.PORT || 8008;
+var PRODUCTION = (process.env.NODE_ENV === 'production');
+var DIST_DIR = path.join(__dirname, 'dist');
+
+var indexStatic;
+var router;
+var app = express();
+
+var startProdApp = function() {
+  console.log([
+    'Production mode enabled. Note that "npm install" is assumed to',
+    'have recently been run with NODE_ENV="production". If this is not',
+    'the case, some or all static assets may be out of date.'
+  ].join('\n'));
+  indexStaticWatcher.build(function(err, newIndexStatic) {
+    if (err) {
+      throw err;
+    }
+
+    console.log('Built server-side bundle.');
```

Comment:
> I tried running `NODE_ENV="production" ORIGIN="http://localhost:8008" node app.js` --> made some copy changes on the homepage --> refreshed the page.  However, the changes I just made didn't get reflected on the site.

Oh, that won't work because `NODE_ENV=production` is so optimized for deployment (not development) that it doesn't watch source files or anything--it just compiles the JSX once and then uses it forever, until it's shut down. The only way to have it read the new code is to either reboot the server or use the development mode for the server, which *does* watch the filesystem for changes and rebuilds the bundle when needed.